src/components/AboutUs/index.css

```diff
@@ -1,19 +1,20 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
```

Comment:
No need for this line, since you are the one who added app.css

Reply:
resolve